scripts/lib/install

```diff
@@ -539,3 +539,7 @@ fi
 su zulip -c '/home/zulip/deployments/current/scripts/setup/initialize-database --quiet'
 
 su zulip -c '/home/zulip/deployments/current/manage.py generate_realm_creation_link'
+
+# setting up the gitconfig
+su zulip -c "git config --global user.email $ZULIP_ADMINISTRATOR"
+su zulip -c "git config --global user.name $EXTERNAL_HOST"
```

Comment:
This needs to come before the `exit 0` that may happen on line 536.  Move this up to line 524 and it should be good.